front/src/pods/master/master.component.styles.ts

```diff
@@ -45,6 +45,13 @@ export const containerComponent = css`
   display: block;
 `;
 
+export const lector = css`
+  position: absolute;
```

Comment:
You can use these styles to hide content and get it available only for screen readers:

```
export const screenReaderOnly = css```
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
```
```

[Reference](https://www.w3.org/WAI/tutorials/forms/labels/#hiding-the-label-element)